src/main/java/mate/academy/Main.java

```diff
@@ -39,17 +52,49 @@ public static void main(String[] args) {
         tomorrowMovieSession.setMovie(fastAndFurious);
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
 
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
+        MovieSession todayMovieSession = new MovieSession();
+        todayMovieSession.setCinemaHall(firstCinemaHall);
+        todayMovieSession.setMovie(fastAndFurious);
+        todayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector.getInstance(
+                MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
-        movieSessionService.add(yesterdayMovieSession);
+        movieSessionService.add(todayMovieSession);
 
-        System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
+        System.out.println(movieSessionService.get(todayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector.getInstance(
+                AuthenticationService.class);
+        User loginBob;
+
+        try {
+            authenticationService.register("bob@gmail.com", "password");
+            loginBob = authenticationService.login("bob@gmail.com", "password");
+        } catch (RegistrationException e) {
+            logger.log(Level.SEVERE, "Error during user registration: ", e);
+            throw new RuntimeException("Failed to register a new user."
+                    + " Please check your email or password.", e);
+        } catch (AuthenticationException e) {
+            logger.log(Level.SEVERE, "Error during user authentication: ", e);
+            throw new RuntimeException("Failed to login in account, "
+                    + "email or password is incorrect.");
+        }
+
+        ShoppingCartService shoppingCartService = (ShoppingCartService) injector.getInstance(
+                ShoppingCartService.class);
+        shoppingCartService.addSession(todayMovieSession, loginBob);
+        shoppingCartService.addSession(tomorrowMovieSession, loginBob);
+        System.out.println(shoppingCartService.getByUser(loginBob));
+
+        //Testing the functionality of OrderDao and OrderService.
+        OrderService orderService = (OrderService) injector.getInstance(
+                OrderService.class);
+        //1. Completing a new order.
+        orderService.completeOrder(shoppingCartService.getByUser(loginBob));
+        //2. Retrieving order history for the user.
+        System.out.println(orderService.getOrdersHistory(loginBob));
```

Comment:
Consider replacing `System.out.println` with a logging statement for consistency with the rest of the code, which uses the `Logger` class.